deployment/DeployAgent/roles/install-logstash/tasks/config-gcp-key.yml

```diff
@@ -2,10 +2,17 @@
 - name: Google Pubsub | Copy JSON key file to server
   copy:
     src: "{{ gcp_json_key_file }}"
-    dest: "/tmp/gcp_json_key_file"
+    dest: "/tmp/gcp_json_key_file.json"
 
-- name: Google Pubsub | Set up JSON key file
-  command: >
-    mkdir /etc/logstash/keys
-    mv /tmp/gcp_json_key_file /etc/logstash/keys/gcp_json_key_file
+- name: Google Pubsub | Make key file folder
```

Comment:
does this need the root permission?

Reply:
i haven't had to explicitly set permissions in my testing